mobile/src/bundles/common/components/selector/selector.tsx

```diff
@@ -0,0 +1,114 @@
+    const [isModalVisible, setIsMOdalVisible] = React.useState(false);
+
+    const DropdownButton = React.useRef<View>(null);
+
+    const changeModalVisibility = (): void => {
+        setIsMOdalVisible((previous) => !previous);
+    };
+
+    const onPressItem = (option: Select): void => {
+        changeModalVisibility();
+        setChooseData(option.label);
+    };
+
+    return (
+        <View>
+            <Text
+                category="Label"
+                style={{ fontWeight: '500', color: Color.TEXT }}
+            >
+                {label}
+            </Text>
+            <Pressable
+                ref={DropdownButton}
+                style={[
+                    globalStyles.pv10,
+                    globalStyles.pl15,
+                    globalStyles.borderRadius5,
+                    globalStyles.flexDirectionRow,
+                    globalStyles.justifyContentSpaceBetween,
+                    globalStyles.alignItemsCenter,
+                    styles.dropdownButton,
+                ]}
+                onPress={changeModalVisibility}
+            >
+                <Text
+                    category="Label"
+                    style={{ fontWeight: '400', color: Color.TEXT2 }}
+                >
+                    {chooseData}
+                </Text>
+                <View style={[globalStyles.mr10, globalStyles.p5]}>
+                    {/* <Icon
+                        name="arrow-drop-down"
+                        size={12}
+                        color={Color.PRIMARY}
+                    /> */}
+                </View>
+            </Pressable>
+            {isModalVisible && (
+                <View>
+                    <Pressable
+                        style={[globalStyles.flex1]}
+                        onPress={changeModalVisibility}
+                    >
+                        <View style={[globalStyles.pl20, styles.dropdown]}>
+                            <FlatList
+                                data={options}
+                                renderItem={({ item }): ReactElement => {
+                                    return (
+                                        <TouchableOpacity
+                                            style={globalStyles.pt10}
+                                            onPress={(): void => {
+                                                onPressItem(item);
+                                            }}
+                                        >
+                                            <Text
+                                                category="Label"
+                                                style={globalStyles.pb5}
+                                            >
+                                                {item.label}
+                                            </Text>
+                                        </TouchableOpacity>
+                                    );
+                                }}
```

Comment:
```suggestion
                              
                                        <TouchableOpacity
                                            style={globalStyles.pt10}
                                            aciveOpacity={?}
                                            onPress={(): void => 
                                                onPressItem(item)
                                            }
                                        >
                                            <Text
                                                category="Label"
                                                style={globalStyles.pb5}
                                            >
                                                {item.label}
                                            </Text>
                                        </TouchableOpacity>
                    
                               }
```

Why applying style={globalStyles.pb5} to the text?

Reply:
I don't have a design for the dropdown items. I removed it for now.